tracer/test/Datadog.Trace.Security.Unit.Tests/WafUpdateTests.cs

```diff
@@ -0,0 +1,92 @@
+// <copyright file="WafUpdateTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Linq;
+using Datadog.Trace.AppSec;
+using Datadog.Trace.AppSec.Waf;
+using Datadog.Trace.AppSec.Waf.ReturnTypes.Managed;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.Vendors.Newtonsoft.Json;
+using FluentAssertions;
+using Xunit;
+
+namespace Datadog.Trace.Security.Unit.Tests
+{
+    [Collection("WafTests")]
+    public class WafUpdateTests
+    {
+        public const int TimeoutMicroSeconds = 1_000_000;
+
+        [Theory]
+        [InlineData("[$ne]|arg|crs-942-290", "attack|appscan_fingerprint|crs-913-120")]
+        [InlineData("attack|appscan_fingerprint|crs-913-120", "value|sleep(10)|crs-942-160")]
+        public void RuleToggling(string attack1, string attack2)
```

Comment:
Is there a particular reason you have an `attack1` and `attack2` parameter instead of just a single `attack` parameter? Having a single `attack` parameter might make the unit test simpler if that will still test everything you want

Reply:
This particular is to test the enabling of a single rule. Having only one attack could succeed if the whole security is disabled, for instance, instead of a single rule, which is the purpose of this test.